apps/desktop/src/components/Settings/SettingsModal.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useState, useMemo, useEffect } from 'react';
+import { useTheme } from '../../theme';
+import { getSections } from './schemas';
+import { SettingsSection } from './SettingsRenderer';
+import { XMarkIcon } from '../Icons';
+
+interface SettingsModalProps {
+    onClose: () => void;
+    onResetVault: () => void;
+}
+
+export const SettingsModal: React.FC<SettingsModalProps> = ({ onClose, onResetVault }) => {
+    const { availableThemes } = useTheme();
+    const appVersion = "0.1.0"; // TODO: import from package.json or config
```

Comment:
I noticed there is a TODO here, is it worth doing now? If not then add it to `TODO.md`